docs/ucdxml.md

```diff
@@ -29,6 +31,28 @@ mvn compile exec:java '-Dexec.mainClass="org.unicode.xml.UCDXML"' '-Dexec.args="
 mvn compile exec:java '-Dexec.mainClass="org.unicode.xml.UCDXML"' '-Dexec.args="--range NOUNIHAN --output GROUPED"' -DCLDR_DIR=$(cd ../cldr; pwd) -DUNICODETOOLS_GEN_DIR=$(cd ../Generated; pwd) -DUNICODETOOLS_REPO_DIR=$(pwd)
 ```
 
+### Linux
+
+Including commands for zipping the XML files for publication.
+
+```
+mvn compile exec:java -Dexec.mainClass="org.unicode.xml.UCDXML" -Dexec.args="--range ALL --output FLAT" -DCLDR_DIR=/usr/local/google/home/mscherer/cldr/uni/src -DUNICODETOOLS_GEN_DIR=$(cd ../Generated; pwd) -DUNICODETOOLS_REPO_DIR=$(pwd) -am -pl unicodetools
```

Comment:
"/usr/local/google/home/mscherer/cldr/uni/src" ???

Reply:
I’m fine with documenting « whatever works for Markus » as-is (https://github.com/unicode-org/unicodetools/blob/main/docs/pipeline.md#merge has some idiosyncrasies too, I doubt most people call the unicode-org remote la-vache).
But it does sound like the instructions are slightly more specific than « Linux ».

Reply:
PTAL